gwpy/signal/tests/test_filter_design.py

```diff
@@ -192,6 +192,14 @@ def test_convert_to_digital_fir(example_zpk_fs_tuple):
     assert numpy.allclose(dfilt, signal.bilinear(b, [1], fs))
 
 
+def test_convert_to_digital_complex_type_preserved():
```

Comment:
Can you please add a minimal docstring for the test function, at least with the motivation or a link to an issue (for regression testing)?